CharlieBackend.Api.UnitTes/CurrentUserTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Moq;
+using Xunit;
+using System;
+using System.Security.Claims;
+using System.Security.Principal;
+using Microsoft.AspNetCore.Http;
+using CharlieBackend.Core.Entities;
+using CharlieBackend.Business.Services;
+using FluentAssertions;
+
+
+namespace CharlieBackend.Api.UnitTest
+{
+    public class CurrentUserTests : TestBase
+    {
+        private readonly Mock<IHttpContextAccessor> _httpContextAccessorMock;
+        private readonly DefaultHttpContext _httpContext;
+        private readonly Mock<GenericIdentity> _mockIdentity;
+        private  CurrentUserService _currentuserService;
+
+        public CurrentUserTests()
+        {
+            _httpContextAccessorMock = new Mock<IHttpContextAccessor>();
+            _httpContext = new DefaultHttpContext();
+            _mockIdentity = new Mock<GenericIdentity>("Test");
+            _currentuserService = new CurrentUserService(_httpContextAccessorMock.Object);
+            _httpContext.User = new ClaimsPrincipal(_mockIdentity.Object);
+            _httpContextAccessorMock.Setup(req => req.HttpContext).Returns(_httpContext);
+        }
+
+        [Fact]
+        public void ThrowErrorIfNotLoggined()
+        {
+            _httpContextAccessorMock.Setup(req => req.HttpContext)
+                .Returns(new DefaultHttpContext());
+
+            // Act
+            Func<long> act = () => _currentuserService.AccountId;
+
+            //Assert
+            act.Should().Throw<UnauthorizedAccessException>();
+        }
+
+        [Fact]
+        public void getAccountId()
```

Comment:
Please follow [C# coding conventions](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions). 
When naming public members of types, such as fields, properties, events, methods, and local functions, use pascal casing.